indexer/index.cpp

```diff
@@ -75,6 +76,17 @@ unique_ptr<MwmSet::MwmValueBase> Index::CreateValue(MwmInfo & info) const
   return unique_ptr<MwmSet::MwmValueBase>(move(p));
 }
 
+unique_ptr<FeatureType> Index::GetOriginalFeature(FeatureID const & id) const
```

Comment:
What about edited features?

Reply:
Discussed by voice, I'll add TODO note about that.